base/libgit2/types.jl

```diff
@@ -403,6 +403,44 @@ end
     LibGit2.MergeOptions
 
 Matches the [`git_merge_options`](https://libgit2.github.com/libgit2/#HEAD/type/git_merge_options) struct.
+
+The fields represent:
+  * `version`: version of the struct in use, in case this changes later. For now, always `1`.
+  * `flags`: an `enum` for flags describing merge behavior.
+     Defined in [`git_merge_flag_t`](https://github.com/libgit2/libgit2/blob/HEAD/include/git2/merge.h#L95).
+     The corresponding Julia enum is `GIT_MERGE` and has values:
```

Comment:
Perhaps we should just document the enum directly and then link to it here?

Reply:
I think this is going to be more effort than it's worth. For one, most of these "enums" are not actually Julia enums, they're just constant (integer) values and we'd have to convert all of them for the docstrings to attach. For two, I seriously doubt many people are going to be referring to most of these flags often enough to merit having them in the devdocs and having someone keep those docs up to date. The vast majority of the time people are calling with the default values and if you're not, you're probably able to look up `base/libgit2/consts.jl`. If I had my druthers we wouldn't have any inline docs in `base/libgit2/consts.jl` since I very seriously doubt anyone is looking in there for anything and it's another thing we have to keep from getting desynced.

Reply:
Excellent points, agreed. Never mind me then. 👍